README.md

```diff
@@ -303,6 +304,100 @@ There are some settings that can be configured via a `settings.json` file:
 - VS Code: These settings will show up in the settings window
 - LanguageClient-neovim: Create this file in `$projectdir/.vim/settings.json` or set `g:LanguageClient_settingsPath`
 
+## Explicit Configuration
+
+**For a full explanation of possible configuration, we refer to [hie-bios/README](https://github.com/mpickering/hie-bios/blob/master/README.md).**
+
+The user can place a `hie.yaml` file in the root of the workspace which
+describes how to setup the environment. For example, to explicitly state
+that you want to use `stack` then the configuration file would look like:
+
+```yaml
+cradle: {stack}
+```
+
+If you use `cabal` then you probably need to specify which component you want
+to use.
+
+```yaml
+cradle:
+  cabal:
+    component: "lib:haskell-ide-engine"
+```
+
+Or you can explicitly state the program which should be used to collect
+the options by supplying the path to the program. It is interpreted
+relative to the current working directory if it is not an absolute path.
+
+```yaml
+cradle:
+  bios:
+    program: ".hie-bios"
+```
+
+The complete configuration is a subset of
+
+```yaml
+cradle:
+  cabal:
+    component: "optional component name"
+  stack:
+  bazel:
+  obelisk:
+  bios:
+    program: "program to run"
+    dependency-program: "optional program to run"
+  direct:
+    arguments: ["list","of","ghc","arguments"]
+  default:
+  none:
+
+dependencies:
+  - someDep
+```
```

Comment:
This is out of date.

Reply:
why? Maybe I am overlooking something but in hie-bios, it looks the same: https://github.com/mpickering/hie-bios#cradle-dependencies

>
> ```yaml
> cradle:
>  cabal:
>    component: "lib:hie-bios"
> dependencies:
>  - package.yaml
>  - shell.nix
>  - default.nix
> ```

Reply:
The obelisk cradle doesn't exist anymore for example.

Reply:
oh, i see, I thought you meant the dependencies. Obelisk and bazel cradle has been removed in mpickering#90